lib/github_changelog_generator/parser_file.rb

```diff
@@ -66,8 +66,10 @@ def extract_pair(line)
       [key.tr("-", "_").to_sym, value.gsub(/[\n\r]+/, "")]
     end
 
-    KNOWN_ARRAY_KEYS = %i[exclude_labels include_labels bug_labels
-                          enhancement_labels breaking_labels issue_line_labels between_tags exclude_tags]
+    KNOWN_ARRAY_KEYS = %i[exclude_labels include_labels
+                          breaking_labels enhancement_labels bug_labels
+                          deprecated_labels removed_labels security_labels
+                          issue_line_labels between_tags exclude_tags]
```

Comment:
Tiny thing: thanks for keeping the lines short - Github diffs are rendered much better when the lines are shorter (~ 80 chars). Kudos!